recipes/joltphysics/all/conanfile.py

```diff
@@ -133,6 +133,8 @@ def generate(self):
         tc.variables["JPH_PROFILE_ENABLED"] = self.options.profile
         if Version(self.version) >= "3.0.0":
             tc.variables["ENABLE_ALL_WARNINGS"] = False
+        if Version(self.version) >= "3.0.0":
```

Comment:
`3.0.0`? Or `4.0.0`?

Reply:
@RubenRBS 
`INTERPROCEDURAL_OPTIMIZATION` has been introduced since 3.0.0.
I merged these version checks.
Thank you for your review!